resources/systemd/openhab2.service

```diff
@@ -6,13 +6,17 @@ Wants=network-online.target
 After=network-online.target
 
 [Service]
-EnvironmentFile=/etc/default/openhab2
-EnvironmentFile=/etc/default/openhab2_dirs
+Environment=OPENHAB_HOME=/usr/share/openhab2
+Environment=OPENHAB_CONF=/etc/openhab2
+Environment=OPENHAB_RUNTIME=/usr/share/openhab2/runtime
+Environment=OPENHAB_USERDATA=/var/lib/openhab2
+Environment=OPENHAB_LOGDIR=/var/log/openhab2
+Environment=OPENHAB_USER=openhab
+Environment=OPENHAB_GROUP=openhab
+EnvironmentFile=-/etc/default/openhab2
 User=openhab
 Group=openhab
```

Comment:
Despite the efforts of this PR, the service file needs a static user and group. If systemd users want to run the service as a different user or group they'll have to specify then again here.